Source/SwiftLintFramework/Rules/UntypedErrorRule.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  UntypedErrorRule.swift
+//  SwiftLint
+//
+//  Created by Daniel.Metzing on 17/02/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct UntypedErrorRule: ASTRule, OptInRule, ConfigurationProviderRule {
```

Comment:
What do you think about changing it to `UntypedErrorInCatchRule`

Reply:
And then chaning the rule identifier and name as well

Reply:
Updated the naming as you suggested, its for sure more clear what does it.